### PR TITLE
integration-tests: update integration tests for new snap.yaml

### PR DESCRIPTION
**Pull request by mvo5**
Also skip some of the failover tests for now that look like they
are broken for unreleated reasons.

### CODE REVIEW

File: integration-tests/data/snaps/basic-binaries/meta/snap.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ echo:
+  command: bin/echo
+  uses: [nothing]
+ success:
+  command: bin/success
+  uses: [nothing]
+ fail:
+  command: bin/fail
+  uses: [nothing]
+uses:
```
Comment (zyga) [inline integration-tests/data/snaps/basic-binaries/meta/snap.yaml]: What is the point of the migration skill here?

**Comment by zyga** [comment]
+1, just a question on the use of migration skill

File: integration-tests/data/snaps/basic-binaries/meta/snap.yaml (hunk 1 above)
Comment (mvo5) [inline integration-tests/data/snaps/basic-binaries/meta/snap.yaml]: The default permissions for apps are `caps: [network-client]`. This is giving the basic binaries less permissions than this. But yeah, we don't really need this, we could use the default permissions. But note that we currently do not add the migration-skill by default. Maybe we should during the transition period.

File: integration-tests/tests/failover_rclocal_crash_test.go [hunk 2]
```diff
@@ -29,6 +29,8 @@ import (
 )
 
 func (s *failoverSuite) TestRCLocalCrash(c *check.C) {
+	c.Skip("FIXME: fails for unknown reasons")
```
Comment (fgimenez) [inline integration-tests/tests/failover_rclocal_crash_test.go]: The three failover tests are passing locally for me with the changes in this branch and the latest udf [1], could you please check again?

[1] http://paste.ubuntu.com/14856762/

**Comment by fgimenez** [comment]
Great job, thanks! :) :+1: only concern about the failover tests being skipped

**Comment by mvo5** [comment]
@fgimenez excellent, I re-enabled the failover tests again, that was probably just a local issue then. Its failing right now because of some store issue that looks unreleated to what we do.

**Comment by mvo5** [comment]
The fix for the store issue is part of bugfix/bugfix/stop-sending-15.04-dev-framework

**Pull request merged**